src/components/TeamBio/TeamBio.stories.tsx

```diff
@@ -4,40 +4,51 @@ import { ComponentStory, ComponentMeta } from "@storybook/react";
 import TeamBio from "./TeamBio";
 
 export default {
-    title: "React Component Library/Team Bio",
-    component: TeamBio,
+  title: "React Component Library/Team Bio",
```

Comment:
Can you create a folder for Team Bio and put everything related to in in that folder? Something like 
"React Component Library/Team Bio/Team Bio"?